showcase/app/routes/page-components/app-side-nav/index.ts

```diff
@@ -0,0 +1,12 @@
+/**
```

Comment:
I see that `ComponentsAppSideNavModel` (similar for `PageComponentsAppHeaderModel` above) is never used, but is declared anyway. Does it mean that in the future for every component we have to add this block of code (for consistency, I imagine?). In that case, do you think is worth to update our blueprint script to account for this new code pattern?

Reply:
We don't have to add it if its not used... I just thought it would be good for consistency. I can definitely update the blueprint script after this is merged (itd be easier to review that change just on its own)